src/components/ResourceList/ResourceList.tsx

```diff
@@ -343,8 +345,7 @@ export class ResourceList extends React.Component<CombinedProps, State> {
       onSortChange,
       polaris: {intl},
     } = this.props;
-    const {selectMode, loadingPosition, listNode} = this.state;
-    const itemsExist = items.length > 0;
```

Comment:
I see. I think I introduced this bug when adding `loading`. Sorry about that.

Reply:
No worries 😄